ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettySpecificTest.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.server.netty
+
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import org.junit.*
+
+class NettySpecificTest {
```

Comment:
Class name?

Reply:
That's a good name for the test, but not for the suite I guess